pkg/infoschema/tables.go

```diff
@@ -1663,6 +1666,22 @@ var tableKeywords = []columnInfo{
 	{name: "RESERVED", tp: mysql.TypeLong, size: 11},
 }
 
+var tableTiDBParamsCols = []columnInfo{
+	{name: "COMPONENT", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "TYPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "NAME", tp: mysql.TypeVarchar, size: 256, flag: mysql.NotNullFlag},
+	{name: "INSTANCE", tp: mysql.TypeVarchar, size: 64},
+	{name: "SCOPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
```

Comment:
```
sql> SELECT SCOPE, COUNT(*) FROM information_schema.TIDB_PARAMS GROUP BY SCOPE;
+----------------+----------+
| SCOPE          | COUNT(*) |
+----------------+----------+
| INSTANCE       |      219 |
| SESSION,GLOBAL |      311 |
| NONE           |      153 |
| GLOBAL         |      334 |
| SESSION        |       57 |
+----------------+----------+
5 rows in set, 1 warning (0.0177 sec)
Warning (code 1105): tikv node store1 does not contain status address

sql> SELECT COUNT(*) FROM information_schema.TIDB_PARAMS WHERE FIND_IN_SET('GLOBAL', SCOPE);
+----------+
| COUNT(*) |
+----------+
|      645 |
+----------+
1 row in set, 1 warning (0.0226 sec)
Warning (code 1105): tikv node store1 does not contain status address

```

We could make this a [`SET`](https://docs.pingcap.com/tidb/stable/data-type-string#set-type) type and then allow the use of [`FIND_IN_SET()`](https://docs.pingcap.com/tidb/stable/string-functions) for this. However even with the current implementation this seems to work.

Or we could use a JSON ARRAY for this:
```
sql> WITH a AS (SELECT JSON_ARRAY("GLOBAL", "SESSION") AS SCOPE) SELECT * FROM a WHERE "GLOBAL" MEMBER OF (SCOPE);
+-----------------------+
| SCOPE                 |
+-----------------------+
| ["GLOBAL", "SESSION"] |
+-----------------------+
1 row in set (0.0014 sec)
```
Then we could use [`MEMBER OF`](https://docs.pingcap.com/tidb/stable/json-functions#functions-that-search-json-values).

Reply:
@dveeden Thanks for your comments. I will use `SET` type instead of `VarChar`.